src/modules/nasa/infra/t-temporal-api.ts

```diff
@@ -0,0 +1,46 @@
+export type TTimeSeriesData = {
+  [date: string]: number; // key is the date, value is the measurement for that parameter
+};
+
+export type TClimateParameters = {
+  T2M: TTimeSeriesData; // Temperature at 2 meters (°C)
+  TS: TTimeSeriesData; // Earth Skin Temperature (°C)
+  PRECTOTCORR: TTimeSeriesData; // Precipitation (mm/day)
+  QV2M: TTimeSeriesData; // Specific Humidity at 2 meters (g/kg)
+  WS2M: TTimeSeriesData; // Wind Speed at 2 meters (m/s)
+  GWETTOP: TTimeSeriesData; // Surface Soil Wetness (1)
+  GWETROOT: TTimeSeriesData; // Root Zone Soil Wetness (1)
+  GWETPROF: TTimeSeriesData; // Profile Soil Moisture (1)
+};
+
+export type TClimateData = {
+  type: string; // Feature type, e.g., "Feature"
+  geometry: {
+    type: string; // Geometry type, e.g., "Point"
+    coordinates: [number, number, number]; // Longitude, Latitude, and Elevation
```

Comment:
The type system can make this clearer 

```suggestion
    coordinates: [longitude: number, latitude: number, elevation: number];
```